source/E2E.Tests/Services/MobileParties/BanditPartyComponentTests.cs

```diff
@@ -72,4 +74,60 @@ public void ClientCreateParty_DoesNothing()
             Assert.False(client.ObjectManager.TryGetObject<MobileParty>(partyId, out var _));
         }
     }
+
+    [Fact]
+    public void ClientUpdateParty_DoesNothing()
+    {
+        // Arrange
+        var server = TestEnvironment.Server;
+        var client1 = TestEnvironment.Clients.First();
+
+        var component = GameObjectCreator.CreateInitializedObject<BanditPartyComponent>();
+        var hideout = GameObjectCreator.CreateInitializedObject<Hideout>();
+        var settlement = GameObjectCreator.CreateInitializedObject<Settlement>();
```

Comment:
This isn't guaranteed to run on the server, could you add this in a server.call statement?